lib/ClangImporter/ImportDecl.cpp

```diff
@@ -7048,7 +7048,7 @@ void SwiftDeclConverter::importInheritedConstructors(
 
   auto curObjCClass = cast<clang::ObjCInterfaceDecl>(classDecl->getClangDecl());
 
-  auto inheritConstructors = [&](ArrayRef<ValueDecl *> members,
+  auto inheritConstructors = [&](TinyPtrVector<ValueDecl *> members,
```

Comment:
This one should be okay, no? Passing something down the stack should always be okay (because of C++'s "destruction happens at the end of the full-expression" rule.)

Reply:
ASan didn’t show any failures here—I just thought it looked suspicious and would be fairly cheap to recode defensively—so if that’s the rule, this one is probably unnecessary.